packages/react-native-renderer/src/ReactNativeFiberHostComponent.js

```diff
@@ -124,6 +125,14 @@ class ReactNativeFiberHostComponent {
       );
     }
   }
+
+  sendAccessibilityEvent(eventType: string) {
+    if (eventType === 'focus') {
+      DeprecatedAccessibilityInfo.setAccessibilityFocus(this._nativeTag);
+    } else {
```

Comment:
If Fabric is not going to throw on invalid strings from JS, we probably want that behavior consistent on paper. Probably just no-op. Enforcement can come from the types